File: Assets/Smash Way/Scripts CS/GameOther/ControlAll.cs
using UnityEngine;
using System.Collections;

public class ControlAll : MonoBehaviour {


	public bool GravityTrue=false;
	public bool GravityFalse=false;
	public bool GravityAnti=false;
	public bool GravityDefault=false;
	public float GravityDefaultX;
	public float GravityDefaultY;
	public float GravityDefaultZ;
	//var Cam:GameObject;


	void Awake () {
		if(GravityTrue){Physics.gravity= new Vector3 (0,-9.81f,0);}
		if(GravityFalse){Physics.gravity= new Vector3 (0,0,0);}
		if(GravityAnti){Physics.gravity= new Vector3 (0,9.81f,0);}
		if(GravityDefault){
			Physics.gravity = new Vector3 (GravityDefaultX,GravityDefaultY,GravityDefaultZ);
		}
	}

}

File: Assets/Smash Way/Scripts JS/GameOther/ControlAll.js
#pragma strict
@ ImageEffectOpaque

var GravityTrue:boolean=false;
var GravityFalse:boolean=false;
var GravityAnti:boolean=false;
var GravityDefault:boolean=false;
var GravityDefaultX:float;
var GravityDefaultY:float;
var GravityDefaultZ:float;
//var Cam:GameObject;


function Awake () {
if(GravityTrue){Physics.gravity.y=-9.81;Physics.gravity.x=0;Physics.gravity.z=0;}
if(GravityFalse){Physics.gravity.y=0;Physics.gravity.x=0;Physics.gravity.z=0;}
if(GravityAnti){Physics.gravity.y=9.81;Physics.gravity.x=0;Physics.gravity.z=0;}
if(GravityDefault){
Physics.gravity.x=GravityDefaultX;
Physics.gravity.y=GravityDefaultY;
Physics.gravity.z=GravityDefaultZ;
}
}
